percona-packaging/redhat/percona-server-mongodb.spec.template

```diff
@@ -292,8 +292,14 @@ if [[ $AUTH_ENABLED == 0 ]]; then
     echo " ** Read and write access to data and configuration is unrestricted."
     echo " ** To fix this please use /usr/bin/percona-server-mongodb-enable-auth.sh "
 fi
-
-
+if [ $1 -gt 1 ]; then
+%if 0%{?systemd}
+    %systemd_post_with_restart
+%else
+if [ -x %{_sysconfdir}/init.d/mongod ] ; then
+  %{_sysconfdir}/init.d/mongod restart
```

Comment:
Will this restart service even if it wasn't started before or just in case it was already started before upgrade?